lib/screens/dashboard/dashboard.dart

```diff
@@ -1,6 +1,6 @@
 import 'package:ecochat_app/models/marker_model.dart';
 import 'package:flutter/material.dart';
-import 'package:ecochat_app/services/signal_r.dart';
+import 'package:ecochat_app/services/signalr_markers.dart';
```

Comment:
kan het niet beter "markers_signalr.dart" worden aangezien we ook "markers_api.dart" hebben